README.md

```diff
@@ -1,12 +1,15 @@
-# Kf
+# Migrate Cloud Foundry applications to Kubernetes using Kf
+As your teams standardize on Kubernetes, migrating applications from existing platforms like Cloud Foundry is often one of the biggest challenges. Kf, which is now offered as a fully managed service by [Google Cloud Platform](https://cloud.google.com), was designed to help your teams minimize any disruption to developer workflows during the migration to [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) and [Anthos](https://cloud.google.com/anthos). 
```

Comment:
I hesitate to say Kf is a "fully managed service" when it's something customers install onto K8s clusters they have to run and maintain. In the CF world, fully managed sets different expectations (more akin to serverless).

Reply:
Yep, I definitely went to far with that - it should say "fully supported"